amethyst_assets/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "amethyst_assets"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Let's also use this version when specifying `amethyst_assets` as dependency in our core crate.